web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -68,82 +70,137 @@ describe("<SidebarCardsList />", () => {
     render(<SidebarCardsList {...sidebarCardsListProps} />);
   };
 
-  it("fire unhide_cards analytics when accordion is opened when displayFundings is true", () => {
-    render(
-      <SidebarCardsList
-        topCards={[]}
-        bottomCards={[]}
-        fundings={[]}
-        hiddenFundings={[]}
-        certifications={[]}
-        hiddenCertifications={[]}
-        displayFundings={true}
-        displayCertifications={false}
-      />
-    );
+  it("fires unhide_cards analytics when accordion is opened when displayFundings is true", () => {
+    renderComponent({ displayFundings: true });
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
 
-  it("fire unhide_cards analytics when accordion is opened when displayCertifications is true", () => {
-    render(
-      <SidebarCardsList
-        topCards={[]}
-        bottomCards={[]}
-        fundings={[]}
-        hiddenFundings={[]}
-        certifications={[]}
-        hiddenCertifications={[]}
-        displayFundings={false}
-        displayCertifications={true}
-      />
-    );
+  it("fires unhide_cards analytics when accordion is opened when displayCertifications is true", () => {
+    renderComponent({ displayCertifications: true });
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
 
-  describe("Empty State Messaging", () => {
-    it("displays Complete Required Tasks Message when there are no top or bottom cards and display Fundings and Certifications are false", () => {
-      renderComponent({});
-      expect(screen.getByTestId("complete-required-tasks-msg")).toBeInTheDocument();
-      expect(screen.queryByText(Config.dashboardDefaults.emptyOpportunitiesText)).not.toBeInTheDocument();
-    });
+  describe("Empty State Messages", () => {
+    describe("Remote Seller/Worker Message", () => {
+      const foreignBusinessTypeIds: ForeignBusinessTypeId[] = [
+        "employeesInNJ",
+        "revenueInNJ",
+        "transactionsInNJ",
+      ];
+      const randomIndex = Math.floor(Math.random() * foreignBusinessTypeIds.length);
```

Comment:
pebble - as mentioned earlier I think it's better to test against all use cases

Reply:
Included